.github/workflows/publish.yml

```diff
@@ -0,0 +1,143 @@
+name: Publish Version
+
+on:
+  workflow_call:
+    inputs:
+      simulate:
+        required: false
+        type: boolean
+        default: true
+      branch:
+        description: 'Branch to checkout; 1.20.1 or 1.21'
+        required: true
+        type: string
+      tag-name:
+        description: 'Tag to upload to'
+        required: true
+        type: string
+      release-body:
+        description: 'Body for published release notes'
+        required: false
+        type: string
+      changelog-body:
+        description: 'Body for CHANGELOG.md'
```

Comment:
wait do we need to copypaste the entire changelog here or?

Reply:
No, the parameter is the set of change notes to be inserted into `CHANGELOG.md`, see L136, it gets inserted at line 3 in the file for the PR. I will make the description clearer.